### PR TITLE
Open virtual link in a new window

### PR DESCRIPTION
**Pull request by wilr**
As opening it in the same window - when going back, displays a broken page.

### CODE REVIEW

File: code/models/ElementVirtualLinked.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ public function __construct($record = null, $isSingleton = false, $model = null)
     public function getCMSFields()
     {
         $message = sprintf(
-            '<p>%s</p><p><a href="%2$s">%2$s</a></p>',
+            '<p>%s</p><p><a href="%2$s" target="_blank">%2$s</a></p>',
```
Comment (ScopeyNZ) [inline code/models/ElementVirtualLinked.php]: I doubt this'll ever be tabnab-able but do you want to add in `rel="noopener noreferrer"` just in case?

**Comment by ScopeyNZ** [comment]
Something funky is happening with the tests :suspect: 

They're "passing" but it doesn't appear as though they actually ran any tests...

**Comment by ScopeyNZ** [comment]
@wilr Did you want to make a PR to the SS4 version?:

https://github.com/dnadesign/silverstripe-elemental-virtual/blob/master/src/Model/ElementVirtual.php#L63

**Comment by wilr** [comment]
@ScopeyNZ can do, this project is SS3 only at the moment but will be upgrading at some point in the year!

**Pull request merged**